src/frontend/App/typer/kanOppretteRevurdering.ts

```diff
@@ -0,0 +1,10 @@
+export type KanOppretteRevurdering =
+    | {
+          kanOpprettes: true;
+      }
+    | { kanOpprettes: false; årsak?: KanIkkeOppretteRevurderingÅrsak };
+
```

Comment:
```suggestion
{
kanOpprettes: boolean;
årsak?: KanIkkeOppretteRevurderingÅrsak;
}
```

Reply:
For typing blir det mer riktigt sånn som jeg gjort det :) då vet man at årsak finnes hvis kanOpprettes er satt til false, og at den ikke finnes hvis kanOpprettes er satt til true